simulation_parameters/Constants.cs

```diff
@@ -397,10 +398,11 @@ public static class Constants
     // Auto evo population algorithm tweak variables
     public const float AUTO_EVO_ATP_USE_SCORE_DIVISOR = 300;
     public const float AUTO_EVO_GLUCOSE_USE_SCORE_DIVISOR = 1;
-    public const float AUTO_EVO_PILUS_PREDATION_SCORE = 1;
-    public const float AUTO_EVO_TOXIN_PREDATION_SCORE = 1;
-    public const float AUTO_EVO_PREDATION_ENERGY_MULTIPLIER = 0.5f;
-    public const float AUTO_EVO_SUNLIGHT_ENERGY_AMOUNT = 6000;
+    public const float AUTO_EVO_ENGULF_PREDATION_SCORE = 100;
+    public const float AUTO_EVO_PILUS_PREDATION_SCORE = 20;
+    public const float AUTO_EVO_TOXIN_PREDATION_SCORE = 10;
+    public const float AUTO_EVO_PREDATION_ENERGY_MULTIPLIER = 0.4f;
+    public const float AUTO_EVO_SUNLIGHT_ENERGY_AMOUNT = 600000;
```

Comment:
This number really represents a population capacity by another name (since you don't know how much energy a single organism will need to survive). Or at least a conversion of sunlight amount in the patch to population density. In the little arena the player swims around in, I don't know how that would reflect, if at all.

Reply:
Well, I'd say it would be great to be able to make a bridge between the two worlds, but I get it might be pretty specific to the current game state.